packages/component-library/icons/ai.icon.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M12 0L13.7536 6.24638L20 8L13.7536 9.75362L12 16L10.2464 9.75362L4 8L10.2464 6.24638L12 0Z" fill="black"/>
```

Comment:
Should it be the hex?
```suggestion
  <path d="M12 0L13.7536 6.24638L20 8L13.7536 9.75362L12 16L10.2464 9.75362L4 8L10.2464 6.24638L12 0Z" fill="#000000"/>
```

Reply:
It can be either, we still have a lot that use fill="black"
![Screenshot 2023-10-06 at 9 01 55 am](https://github.com/cultureamp/kaizen-legacy/assets/36558508/99284b55-00b5-40f7-a40e-cf9afaa438fa)